set-up-page-generation-tutorial/mesh.yml

```diff
@@ -14,26 +14,53 @@ sources:
       - "rendering-contexts"
 
 processing:
-  relay:
+  relay-pages:
     image: relay-processing-service
     incoming:
-      pages:
+      messages:
         topic: inboxes/pages
-      fragments:
-        topic: inboxes/fragments
-      assets:
-        topic: inboxes/assets
-      web-resources:
-        topic: inboxes/web-resources
     outgoing:
-      relayed-pages:
+      relayed-messages:
         topic: outboxes/pages
-      relayed-fragments:
+    environment:
+      MP_MESSAGING_INCOMING_MESSAGES_SCHEMA: page-schema
+      MP_MESSAGING_OUTGOING_RELAYED-MESSAGES_SCHEMA: page-schema
+
+  relay-fragments:
+    image: relay-processing-service
+    incoming:
+      messages:
+        topic: inboxes/fragments
+    outgoing:
+      relayed-messages:
         topic: outboxes/fragments
-      relayed-assets:
+    environment:
+      MP_MESSAGING_INCOMING_MESSAGES_SCHEMA: fragment-schema
+      MP_MESSAGING_OUTGOING_RELAYED-MESSAGES_SCHEMA: fragment-schema
+
+  relay-assets:
+    image: relay-processing-service
+    incoming:
+      messages:
+        topic: inboxes/assets
+    outgoing:
+      relayed-messages:
         topic: outboxes/assets
-      relayed-web-resources:
+    environment:
+      MP_MESSAGING_INCOMING_MESSAGES_SCHEMA: asset-schema
+      MP_MESSAGING_OUTGOING_RELAYED-MESSAGES_SCHEMA: asset-schema
+
+  relay-web-resources:
+    image: relay-processing-service
+    incoming:
+      messages:
+        topic: inboxes/web-resources
+    outgoing:
+      relayed-messages:
         topic: outboxes/web-resources
+    environment:
+      MP_MESSAGING_INCOMING_MESSAGES_SCHEMA: web-resource-schema
+      MP_MESSAGING_OUTGOING_RELAYED-MESSAGES_SCHEMA: web-resource-schema
 
   rendering-engine-processing-service:
     image: rendering-engine-processing-service
```

Comment:
Container of this service fails to run

Reply:
remove outhoing as the serivce was changed:
 compositions:
        topic: inboxes/compositions